invitations/models.py

```diff
@@ -17,27 +17,46 @@
 
 
 @python_2_unicode_compatible
-class Invitation(models.Model):
-
-    email = models.EmailField(unique=True, verbose_name=_('e-mail address'),
-                              max_length=app_settings.EMAIL_MAX_LENGTH)
+class BaseInvitation(models.Model):
```

Comment:
I think you want this in a different file, otherwise Django will automatically create the `Invitation` model in your database anyway. If you remove the app from `INSTALLED_APPS` (which is normal when using swappable models like this...) then when you import this file Django will complain that there's a model *not* in an installed app. This is why `django.contrib.auth` has it's [`AbstractBaseUser` in a different file](https://github.com/django/django/blob/master/django/contrib/auth/base_user.py).

(Also, I'd suggest calling this AbstractBaseInvitation to follow their naming scheme. 👍 )